server.ts

```diff
@@ -0,0 +1,32 @@
+import { createServer } from 'http';
+import { parse } from 'url';
+import { readFileSync } from 'fs';
+// import { paths } from './src/utils/config';
```

Comment:
If you can’t get this to work feel free to remove it for the hardcoded path instead. I can look in to it as a follow up.